portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -18,11 +18,13 @@
 public final class Comment {
 
   private final long id;
+  private final String title;
```

Comment:
What is title used for? Consider adding a comment here

Is title related to this change?

Reply:
I added this commit early in the design process but never further developed the feature in this PR. I've reverted the commit now 👍 https://github.com/thomasogara/step-repository/pull/28/commits/4cc2e465845c67a2c9c97f840f5baa056694ec41